README.md

```diff
@@ -1 +1,5 @@
 # team-portfolio-gyan-mates
```

Comment:
@Saad-ahamed-m  Do we really need this here ? It should be present for the people who see your website. 

Add a div and please put this message there. I was blank when I saw your website.

Have a read me for this project `Team-Portfolio-Gyan-Mates`

Reply:
Yeah I had added in the login page